podcasts/OnlineSupportController.swift

```diff
@@ -61,6 +64,11 @@ class OnlineSupportController: PCViewController, WKNavigationDelegate {
         dismiss(animated: true, completion: nil)
     }
 
+    @objc private func showStatusPage() {
+        let hostingController = UIHostingController(rootView: StatusPageView().setupDefaultEnvironment())
```

Comment:
We can use the `ThemedHostingController` here.

```suggestion
        let hostingController = ThemedHostingController(rootView: StatusPageView())
```